src/variables.tf

```
@@ -40,6 +40,16 @@ variable "master_password" {
   description = "(Required unless a snapshot_identifier is provided) Password for the master DB user. Note that this may show up in logs, and it will be stored in the state file. Please refer to the DocumentDB Naming Constraints"
 }
 
+variable "manage_master_user_password" {
+  type        = bool
+  description = "Whether to manage the master user password using AWS Secrets Manager."
+  default     = null
```

Why can it not be `false`? 
```suggestion
  default     = false
  nullable    = false
```

---

@goruha This is an oddity because the module parameter only accepts `true` and `null` from what I can see. 

https://github.com/cloudposse/terraform-aws-documentdb-cluster/blob/9e70997acb51b1b8ebdfb2dbc982b95e4c3e3b9a/variables.tf#L102

---

That's weird. But fine.